.gitignore

```diff
@@ -4,3 +4,4 @@
 build/
 gradle/
 lib/
+out
```

Comment:
It is a good practice to add the new lines at the end of file

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline